tests/test_parsing.py

```diff
@@ -274,3 +299,11 @@ def test_filter_top_level_names():
         'to_name': '',
     }
     assert list(filter_top_level_names([dependency])) == [expected]
+
+
+@pytest.mark.parametrize('src,module_version', [
+    ('print "python27", 1000L', ModuleAST27),
+    ('print("python3")', ModuleAST3),
```

Comment:
If there's really a need for differentiating between Python 3 minor versions in `_parse_ast`, we should test for them.